### DISCUSSION TITLE
ACCESS 笔记组织法

### DISCUSSION THREAD

**Discussion by giscus** [oldwinter/knowledge-garden#31] [主发布站的评论系统]
# ACCESS 笔记组织法

Folders or Links? The ACCESS Approach - YouTube
出自 [[LYT 笔记框架]]，根据自身情况，做了小小的改变。我的理解和使用方法如下：
理解 [[🧑Nick Milo]]为了解决 [[PKM系统的不同侧重点-四象限图 by lyt]] 中提到的 3 对矛盾，提出了 ACCESS 文件夹结构。

https://garden.oldwinter.top/ACCESS-%E7%AC%94%E8%AE%B0%E7%BB%84%E7%BB%87%E6%B3%95/

**Comment by cynial** [comment]
实际还是分了很多文件夹，要按文件夹做分类，没有脱离以前的知识管理做法，是一个折中的方案

**Comment by vipyanzg** [comment]
您在Calendar里面的笔记产生的待分类的笔记在分类的时候——比如，移动到Cards文件夹中的时候，是如何操作的？

**Reply by oldwinter** [reply]
看心情，无法归类的片段式笔记 就先通通丢到cards里。其他的放到文件夹归类好。
